wandb/sdk/wandb_run.py

```diff
@@ -3668,24 +3668,33 @@ def finish(exit_code: Optional[int] = None, quiet: Optional[bool] = None) -> Non
         wandb.run.finish(exit_code=exit_code, quiet=quiet)
 
 
+class InvalidArtifact:
```

Comment:
Not related to this PR, but i was thinking about the organization of the `wandb_run` file and curious what do you think about moving the artifact related class to a saperate file closer to other artifact files?